main/src/ui/util/helper.vala

```diff
@@ -299,9 +299,9 @@ public static string parse_add_markup(string s_, string? highlight_word, bool pa
                     int start, end;
                     match_info.fetch_pos(2, out start, out end);
                     return parse_add_markup(s[0:start-1], highlight_word, parse_links, parse_text_markup, already_escaped) +
-                        "<span color='#999'>" +  s[start-1:start] + "</span>" +
+                        "<span alpha='30%'>" +  s[start-1:start] + "</span>" +
```

Comment:
Wasn't the color previously more like 40%? I think that 30% might be too light for some users and they might not see them.

Reply:
@minils any update on this?